airflow/ui/src/components/ClearRun/ClearRunButton.tsx

```diff
@@ -0,0 +1,81 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Box, useDisclosure } from "@chakra-ui/react";
+import { useState } from "react";
+import { FiRefreshCw } from "react-icons/fi";
+
+import type { TaskInstanceCollectionResponse } from "openapi/requests/types.gen";
+import { Button } from "src/components/ui";
+import { useClearDagRun } from "src/queries/useClearRun";
+
+import ClearRunDialog from "./ClearRunDialog";
+
+type Props = {
+  readonly dagId: string;
+  readonly dagRunId: string;
+};
+
+const ClearRunButton = ({ dagId, dagRunId }: Props) => {
+  const { onClose, onOpen, open } = useDisclosure();
+
+  const [onlyFailed, setOnlyFailed] = useState(false);
+
+  const [affectedTasks, setAffectedTasks] =
+    useState<TaskInstanceCollectionResponse | null>();
```

Comment:
```suggestion
    useState<TaskInstanceCollectionResponse | undefined>();
```

Then we don't need to us `as` later.

Reply:
Done thanks.